pkgs/development/libraries/quarto/default.nix

```diff
@@ -11,14 +11,18 @@
 , makeWrapper
 , python3
 , extraPythonPackages ? ps: with ps; []
+
+# Allow overriding the Python and R used by Quarto
+, QUARTO_PYTHON ? "${python3.withPackages (ps: with ps; [ jupyter ipython ] ++ (extraPythonPackages ps))}/bin/python3"
```

Comment:
The name QUARTO_PYTHON should probably be changed e.g. to pythonWithPackages or pythonWrapped, esp as per another comment it should not receive the full path as a string.